libparsec/crates/types/schema/certif/device_certificate.json5

```diff
@@ -41,6 +47,23 @@
                     "discriminant_value": "ED25519"
                 }
             ]
+        },
+        {
+            "name": "DevicePurpose",
+            "variants": [
+                {
+                    "name": "Standard",
+                    "discriminant_value": "STANDARD"
+                },
+                {
+                    "name": "ShamirRecovery",
+                    "discriminant_value": "SHAMIR_RECOVERY"
+                },
+                {
+                    "name": "KeyFileRecovery",
+                    "discriminant_value": "KEY_FILE_RECOVERY"
```

Comment:
I'm thinking if that variant should be called `PassphraseRecovery`, since we do not use a file as is but need a passphrase.

`Shamir` could also work that way (but we don't want that) where the secret that is reconstructed is the key to decrypt the file.

Reply:
I'm ok with both but passphrase sounds a bit better, so let's do that :+1: